src/components/FlowPlayer/FlowPlayer.tsx

```diff
@@ -146,16 +150,18 @@ export const FlowPlayer: FunctionComponent<FlowPlayerProps> = ({
 	}, [start, end]);
 
 	// Draw custom elements
-	flowplayer((opts: any, root: any, api: any) => {
-		const mq = flowplayer.mq;
+	useEffect(() => {
+		flowplayer((opts: any, root: any, api: any) => {
+			const mq = flowplayer.mq;
 
-		api.on('mount', () => {
-			mq('.fp-ui', root).prepend(createTitleOverlay());
-			mq('.fp-ui', root).prepend(createLogoOverlay());
+			api.on('mount', () => {
+				mq('.fp-ui', root).prepend(createTitleOverlay());
```

Comment:
Zet die `createTitleOverlay` en `createLogoOverlay` mee in deze useEffect anders gaat die klagen over dat die missen in de dependency array